### ISSUE TITLE
Pan Pixel Offset and Swipe To Slide not working

### ISSUE THREAD

**Issue by D34thStalker** [aryaxt/iOS-Slide-Menu#213]
I have just integrated this script into my project. I was able to get the Left Menu Controller to appear on button press but I can not seem to get the pixel offset or the "swipe to slide" working.

My project is setting this SlideNavigationController as the root view controller, and then within that, I am presenting a tab bar controller. Inside this tab bar controller , the menu slides in with the press of a button ( as stated above ) but enabling swipe controls and pixel offset changes does not seem to be doing anything.

Any ideas?

Thanks!

**Comment by coolcool1994** [comment]
Yeah I am using the same set up with SlideNavigationController embedding UITabBarController and swipe gestures are not working. I heard JASidePanel can get this to work tho: https://github.com/gotosleep/JASidePanels. I might have to switch to this.

**Comment by coolcool1994** [comment]
I Just got it to work. You simply had to implement below code into the UITabbarController custom class. Make sure you set that class as delegate.

- (BOOL)slideNavigationControllerShouldDisplayLeftMenu
{
        return YES;
}

- (BOOL)slideNavigationControllerShouldDisplayRightMenu
{
	return YES;
}
